hack/get_ocp_versions_for_testing.sh

```diff
@@ -2,7 +2,8 @@
 
 set -euo pipefail
 
-SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+CURRENT_FILE=$(python -c "import os; print(os.path.realpath('$0'))")
+SCRIPT_DIR=$(dirname $CURRENT_FILE)
```

Comment:
```suggestion

```

Reply:
what is your suggestion?

Reply:
If you'll go with `os.path.dirname` you won't need that line